src/kudu/security/security_flags.cc

```diff
@@ -27,14 +27,14 @@ namespace security {
 // below. The DH AES ciphers are not included since we are not configured to
 // use DH key agreement.
 const char* const SecurityDefaults::SecurityDefaults::kDefaultTlsCiphers =
+                                   "TLS_AES_256_GCM_SHA384:"
+                                   "TLS_CHACHA20_POLY1305_SHA256:"
+                                   "TLS_AES_128_GCM_SHA256:"
+                                   "TLS_AES_128_CCM_8_SHA256:"
+                                   "TLS_AES_128_CCM_SHA256:"
                                    "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:"
```

Comment:
Seems like ECDHE-RSA-AES256-GCM-SHA384  is the cipher that is negotiated in common case in prod

Reply:
Yeah, it is commonly negotiated in TLSv1.2.

Reply:
Backed out TLSv1.3 ciphers.